examples/contrib/cratedb_cleanup.py

```
@@ -0,0 +1,102 @@
+import sqlalchemy as sa
+from cratedb_toolkit.model import TableAddress
+
+logger = logging.getLogger(__name__)
+
+
+class DatabaseCleanupTask:
+    """
+    A task definition to clean up temporary tables in a database.
+    """
+
+    def __init__(self, schemas: t.List[str] = None, table_prefixes: t.List[str] = None):
```

Ruff (0.14.10)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)